docs/release_notes/release_notes.md

```diff
@@ -4,84 +4,301 @@ title: Release Notes
 sidebar: sidebars/documentation.html
 ---
 
-# Katello 2.3 (Berliner Weisse) Release Notes
-
-{% if site.version && site.version != 'nightly' %}
-For the full release notes, see the [Changelog](https://github.com/Katello/katello/blob/KATELLO-{{ site.version }}/CHANGELOG.md).
-{% else %}
-For the full release notes, see the [Changelog](https://github.com/Katello/katello/blob/master/CHANGELOG.md).
-{% endif %}
```

Comment:
I removed this because I included the full changelog here, and there's no "major" features/headliner things this release.  What do you think?